shell/platform/tizen/tizen_window_elementary.cc

```diff
@@ -247,7 +247,7 @@ void TizenWindowElementary::RegisterEventHandlers() {
         }
       };
   evas_object_event_callback_add(
-      elm_win_, EVAS_CALLBACK_MOUSE_WHEEL,
+      image_, EVAS_CALLBACK_MOUSE_WHEEL,
       evas_object_callbacks_[EVAS_CALLBACK_MOUSE_WHEEL], this);
 
   evas_object_callbacks_[EVAS_CALLBACK_KEY_DOWN] = [](void* data, Evas* evas,
```

Comment:
Only apply to mouse events but not key events?

Reply:
As far as I know evas_image is not suitable for receiving key events.
I know that the key problem was solved by modifying `GetWindowID()`.
When I tested this PR, the key event did or didn't because of ime...
I don't think this is a problem with event callbacks....